ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1311,7 +1377,30 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		60967E121D76B5AC00863853 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "/usr/bin/python ${PROJECT_DIR}/tools/update_build_version.py ${PROJECT_DIR}/src/ADAL_Internal.h ${PROJECT_DIR}/resources/ios/Framework/Info.plist ${PROJECT_DIR}/resources/ios/ADALiOSBundle/ADALiOSBundle-Info.plist";
```

Comment:
We need to put all the Info.plist files as parameters for the above command, because they are in different paths.

Is it better to put all the Info.plist files in a single folder, such that the build script can locate all the Info.plist files?